vsts.yml

```diff
@@ -1,97 +0,0 @@
-resources:
```

Comment:
We should rename vsts-gn.yml to vsts.gn.  We may want to do that after this PR is merged though.

Reply:
> We may want to do that after this PR is merged though.

➕